src/app/rundown-view/components/rundown-tooltips/video-hover-scrub/video-hover-scrub.component.ts

```diff
@@ -12,15 +11,13 @@ import { TooltipMousePosition } from 'src/app/core/models/tooltips'
 })
 export class VideoHoverScrubComponent implements OnInit, OnDestroy, OnChanges {
   @Input() public filename: string
-  @Input() public tooltipHoverElementWidth: number
-  @Input() public tooltipElementHoverMousePosition?: TooltipMousePosition
+  @Input() public videoLengthInPixels: number
+  @Input() public positionInVideoInPixels: number
```

Comment:
If you merge those three you also move the logic of converting the position of the cursor (which is in pixels) into the "timestamp" of the video depending on how wide is the video element is (which is also in pixels) and how much of the video has already been played.

The above logic would need to be implemented in all consumers of `VideoHoverScrubComponent` which, at the time of writing, is two components.

By not merging the three inputs, we can keep the logic in one place. The place where it is needed. 
The alternative would be to make a wrapping component with the responsibility to convert pixel position into the timestamp in the video, but seems like overkill.

Reply:
Currently the logic for computing the video frame to show is based on how the width of OffsetablePiece is implemented. `playedDurationInMs` is only there because of how OffsetablePiece is rendered. 
MiniShelf does not make use of `playedDurationInMs` because of this. So it is only offsetable piece which would have to do some extra calculating before sending the data to hover scrub component.